data-raw/inputs.R

```diff
@@ -5,7 +5,7 @@ devtools::load_all()
 
 # Source:
 # https://github.com/2DegreesInvesting/tiltIndicator/issues/167#issuecomment-1527223646
-inputs <- read_csv(here("data-raw", "inputs.csv")) |>
+inputs <- read_csv(here("data-raw", "inputs.csv"), col_types="ccccnccc") |>
   select(all_of(c(
```

Comment:
@maurolepore `read_csv` is not reading the column `isic_4digit` as character, hence I have to use the parameter `col_types` to resolve this issue. But, this solution could create problems if the arrangement of columns is different in the real data. I am working to resolve this problem

Reply:
I have resolved this problem in this commit: [https://github.com/2DegreesInvesting/tiltIndicator/pull/261/commits/00f9c696072a8833bf0a7952a4650f2070c8297b](url). Is this solution ok?